godotproject/Objects/Sawblade/SawBladeArea.tscn

```diff
@@ -47,4 +51,6 @@ autoplay = true
 max_distance = 1000.0
 bus = "Sfx"
 
+[node name="CollisionShape2D" type="CollisionShape2D" parent="."]
+shape = SubResource( 3 )
```

Comment:
Should this circle collision be added to the sawblade area? As far as I can tell it won't do anything..

Reply:
Doesn't do anything. Godot Engine gave a warning since it didn't have it included. To be honest we don't need the sawblade area anymore but godot makes re-parenting rood node a pain. I think the only way to do it is manually re-editing the .tscn file or recreating the whole object

Reply:
Ohhh because the root node is an Area2D, it needs to have a collision shape... I see
I think there's an action on the right click menu called "Make Scene Root" which I think you can use to adjust it around and then you can remove the SawBladeArea